inst/make_hexsticker.R

```diff
@@ -1,73 +1,73 @@
 library(hexSticker)
 library(sysfonts)
 library(ggplot2)
+library(dplyr)
 
 # font setup
 font_add_google("Zilla Slab Highlight", "useme")
 
 # make standard plot
 outbreak <- simulate_gillespie(seed = 101)
 
-secondary_dist <- data.table(mu = 1.8, sigma = 0.5)
-class(secondary_dist) <- c(class(secondary_dist), "lognormal_samples")
+secondary_dist <- data.frame(mu = 1.8, sigma = 0.5)
+class(secondary_dist) <- c("lognormal_samples", class(secondary_dist))
 secondary_dist <- add_mean_sd(secondary_dist)
 
 obs <- outbreak |>
   simulate_secondary(
-    meanlog = secondary_dist$meanlog[[1]],
-    sdlog = secondary_dist$sdlog[[1]]
+    meanlog = secondary_dist$mu[[1]],
+    sdlog = secondary_dist$sigma[[1]]
   ) |>
   observe_process()
 
 truncated_obs <- obs |>
-  filter_obs_by_obs_time(obs_time = 25)
+  filter_obs_by_obs_time(obs_time = 25) |>
+  slice_sample(n = 200, replace = FALSE)
 
-truncated_obs <- truncated_obs[sample(seq_len(.N), 200, replace = FALSE)]
+combined_obs <- bind_rows(
+  truncated_obs,
+  mutate(obs, obs_at = max(stime_daily))
+) |>
+  mutate(obs_at = factor(obs_at))
 
-combined_obs <- combine_obs(truncated_obs, obs)
-meanlog <- secondary_dist$meanlog[[1]]
-sdlog <- secondary_dist$sdlog[[1]]
+meanlog <- secondary_dist$mu[[1]]
+sdlog <- secondary_dist$sigma[[1]]
 
-plot <- combined_obs |>
+hex_plot <- combined_obs |>
   ggplot() +
-  aes(x = delay_daily) +
+  aes(x = delay_daily, fill = obs_at) +
   geom_histogram(
-    aes(y = after_stat(density), fill = obs_at),
+    aes(y = after_stat(density)),
     binwidth = 1, position = "dodge"
   ) +
-  lims(x = c(0, 18))
-
-if (!missing(meanlog) && !missing(sdlog)) {
-  plot <- plot +
-    stat_function(
-      fun = dlnorm, args = c(meanlog, sdlog), n = 100,
-      col = "#696767b1"
-    )
-}
-
-# strip out most of the background
-hex_plot <- plot +
-  scale_fill_brewer(palette = "Blues", direction = 1) +
+  lims(x = c(0, 18)) +
+  stat_function(
+    fun = dlnorm, args = c(meanlog, sdlog), n = 100,
+    col = "#696767b1"
+  ) +
+  scale_fill_brewer(palette = "Set2", direction = 1) +
   scale_y_continuous(breaks = NULL) +
   labs(x = "", y = "") +
   theme_void() +
   theme_transparent() +
-  theme(legend.position = "none",
-        panel.background = element_blank())
+  theme(
+    legend.position = "none",
+    panel.background = element_blank()
+  )
 
-# make and save hexsticker
+# Make and save hexsticker
 sticker(
   hex_plot,
   package = "epidist",
   p_size = 23,
   p_color = "#646770",
-  s_x = 1,
-  s_y = 0.85,
-  s_width = 1.3,
-  s_height = 0.75,
```

Comment:
It changes the colour and layout. It was because I needed to refactor that code and then felt like changing it. Agree it should have been put in an issue. No I've not run `usethis::use_logo`

Reply:
Okay cool. I kinda preferred the previous colours but I don't feel strongly. If you run usethis::use_logo happy to merge this vs pulling out into a new PR etc

Reply:
I felt like the colours were hard to distinguish. Do you have colours used in the paper? Happy to switch to those.

Reply:
<img width="133" alt="image" src="https://github.com/user-attachments/assets/45652c5e-c942-4320-b32d-bd6018b2efe1">

Maybe matching this real time and retrospective colours?

Reply:
Actually I think this is how I chose them 
<img width="324" alt="image" src="https://github.com/user-attachments/assets/f701a163-63a7-4865-b8f8-b02be0b27f45">